### PR TITLE
Telemetry for time spent in inbound queue

### PR DESCRIPTION
**Pull request by jatgarg**
Refer: https://github.com/microsoft/FluidFramework/issues/8912
This PR handles the first task of measuring time spent in the inbound queue.

### CODE REVIEW

File: packages/runtime/container-runtime/src/connectionTelemetry.ts [hunk 1]
```diff
@@ -81,6 +84,21 @@ class OpPerfTelemetry {
                 });
             }
         });
+
+        this.deltaManager.inbound.on("push", (message: ISequencedDocumentMessage) => {
+            if (message.sequenceNumber % 1000 === 0) {
```
Comment (anthony-murphy) [inline packages/runtime/container-runtime/src/connectionTelemetry.ts:89]: message.traces will get persisted on the op, and increase the payload size. I think i'd prefer if this was local only. Can't deltaQueue itself just track time until process by default. it should be cheap to do for every op, and then you can occasionally log average

Comment (vladsud) [reply packages/runtime/container-runtime/src/connectionTelemetry.ts:89]: I remember I asked that question and got some answer, but I do not remember what the answer was :)
@tanviraumi, @GaryWilber, is it the case that traces are not removed when ops transition to storage?
I glanced at old, downloaded files and I do not see traces in it.
If they end up in a file, we should remove code in DeltaManager that sticks traces to ops.

That said, I agree with Tony, it feels like even if traces are not preserved in files, it still adds to COGS, so if we can not use them, it's better (including DM layer).

Comment (GaryWilber) [reply packages/runtime/container-runtime/src/connectionTelemetry.ts:89]: @vladsud Push deletes the `traces` property when storing the op in Redis & when persisting them to SPO.

Comment (vladsud) [reply packages/runtime/container-runtime/src/connectionTelemetry.ts:89]: Here is my take:
If using traces gives us more information (I'm looking at server supplied data, like how long it took to process / sequence op, i.e. excluding network time), then I'd take that. And code inspection suggests that's what should happen (search for _service: "alfred"_). But when I run the example against ODSP, I see client putting traces on op, but no traces come out on receiving ops back!

So I see two paths:
1. We figure out why traces does not work at all and fix them. And once they work, we follow up with these two steps:
   - remove code from DM to set traces on EVERY op - this is expensive, sampling is enough, and should be driven completely from runtime.
   - We record not just our own metrics (that we can collect via other means), but also anything that service sticks on traces.
2. We remove any references to traces in our code base and implement what we need directly.

**Comment by jatgarg** [comment]
Abandoning this PR as we will not use traces.

**Pull request closed without merging**